src/libsync/account.h

```diff
@@ -94,6 +94,24 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject
     QString davUser() const;
     void setDavUser(const QString &newDavUser);
 
+    /***
+     * With OC 10 this is the equivalent to the sync root.
+     * With ocis and spaces this will be the default folder containing all spaces.
+     * This function will assert if the sync root is empty.
+     */
+    QString defaultSyncRoot() const;
+
+    /***
+     * Whether we have defaultSyncRoot defined.
+     */
+    bool hasDefaultSyncRoot() const;
+
+    /***
+     * Set defaultSyncRoot and creates the path on the filesystem.
+     * Setting an empty string will have no effect.
```

Comment:
Now I'm confused. Is there a reason to ever call this with an empty string?

Reply:
Loading a legacy account.

Reply:
Loading an account without a defaultSyncRoot